Client/game_sa/CAEVehicleAudioEntitySA.h

```diff
@@ -122,27 +122,74 @@
 #define FUNC_CAEVehicleAudioEntity__Service                                            0x502280
 
 #define VAR_CAEVehicleAudioEntity__s_pPlayerDriver                                     0xB6B990
+#define VAR_CAEVehicleAudioEntity__s_aDefaultVehicleAudioSettings                      0x860AF0
+#define VAR_CAEVehicleAudioEntity__s_aDummyEngineSlots                                 0xB6B9A0
```

Comment:
There's no reason to define preprocessors when you can just easily write
```
// header file
const int MAX_DUMMY_ENGINE_SLOTS = 10;
class CAEVehicleAudioEntitySA 
{
public:
    static DummyEngineSlot(&s_DummyEngineSlots)[MAX_DUMMY_ENGINE_SLOTS];
};
```
```
// .cpp file
DummyEngineSlot(&CAEVehicleAudioEntitySA::s_DummyEngineSlots)[MAX_DUMMY_ENGINE_SLOTS] = *(DummyEngineSlot(*)[MAX_DUMMY_ENGINE_SLOTS])0xB6B9A0;
```

Reply:
I know about `constexpr` (and I use it a lot), but I just wanted to be consistent about using macros.
I might as well just copy paste the address directly then.